.github/CODEOWNERS

```
@@ -11,11 +11,15 @@
 # This also holds true for GitHub teams. Since almost none of our teams have write
 # permissions, you need to list all members of the team with commit access individually.
 
-* @NixOS/documentation-team
+* @alejandrosame @asymmetric @henrik-ch @infinisil @olafklingt @proofconstruction
```

```suggestion
* @NixOS/documentation-reviewers 
```

Just created https://github.com/orgs/nixos/teams/documentation-team

---

Is this discoverable by people who are not in the team? How will they know to tag this team and not the -team one?

---

I guess if somebody uses the teams autocompletion they'll likely pick the -reviewers one instead of the -team one.

---

Now pushed this change